packages/drivers/odsp-driver/src/odspSnapshotParser.ts

```diff
@@ -18,7 +18,7 @@ import { ISnapshotContents } from "./odspUtils";
 function buildHierarchy(flatTree: IOdspSnapshotCommit): api.ISnapshotTree {
     const lookup: { [path: string]: api.ISnapshotTree } = {};
     // id is required for root tree as it will be used to determine the version we loaded from.
-    const root: api.ISnapshotTree = { id: flatTree.id, blobs: {}, commits: {}, trees: {} };
+    const root: api.ISnapshotTree = { id: flatTree.id, blobs: {}, trees: {} };
```

Comment:
Do we use ID anywhere? I think ISnapshotTree.id should be deleted as runtime has no capability to leverage it (there is no API to do anything with it).

Reply:
We do use the ID still.